### ISSUE TITLE
why prior_box_layer has no backpropagation implementation?

### ISSUE THREAD

**Issue by mmmonkeytao** [weiliu89/caffe#25]
I don't quite understand how prior_box_layer performs backpropagation if it is not implemented

**Comment by weiliu89** [comment]
prior_box_layer is only used to generate prior (default) bbox, which is fixed. The location prediction (e.g. *_mbox_loc) is the real prediction (offset on top of prior box), which is backpropagated during training.

**Comment by mmmonkeytao** [comment]
I see. In DetectionOutput layer, the fixed prior bbox is required in the loss caculation between predicted bbox and ground truth bbox.
